src/providers/profile/profile.ts

```diff
@@ -325,8 +325,16 @@ export class ProfileProvider {
         delete walletsGroups['read-only'];
 
         // use wallets name for wallets group name at migration
-        name = `Group ${Object.keys(walletsGroups).indexOf(keyId) + 1}`;
+        name = `Key ${Object.keys(walletsGroups).indexOf(keyId) + 1}`;
         this.setWalletGroupName(keyId, name);
+      } else if (name.match(/^Group \d+/)) {
+        const newName = name.replace('Group', 'Key');
+        this.logger.debug(
+          'Renaming wallets group from ' + name + ' to ' + newName
+        );
+        setTimeout(() => {
+          this.setWalletGroupName(keyId, newName);
+        }, 100);
```

Comment:
Not necessary. Check this PR: https://github.com/bitpay/copay/pull/10197/files#diff-a70e4e6911ddccc9abda5a516441a029R69